web/src/components/Market/SwapTokens/SwapTokens.tsx

```diff
@@ -147,7 +147,7 @@ export function SwapTokens({
     chainId,
   );
   const { data: xDAIBalance = BigInt(0) } = useTokenBalance(account, NATIVE_TOKEN, chainId);
-  const isUseWrappedToken = wxDAIBalance > xDAIBalance && chainId === gnosis.id;
+  const isUseWrappedToken = useMemo(() => wxDAIBalance > xDAIBalance && chainId === gnosis.id, []);
```

Comment:
this useMemo is not needed

Reply:
If before converting, you have more xDAI, but after converting to sDAI, you have less xDAI than wxDAI, then the trade dialog will refresh. The useMemo wrapper is used to prevent that.